__tests__/html2/fluentTheme/defaultFeedback.activity.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
```

Comment:
Apply my comments from the white-label experiences, e.g. add test for dislike/cancel button, etc.